src/docs-developer-guide.rst

```diff
@@ -122,3 +122,188 @@ For example:
   }
 
   /* New section ends */
+
+.. _style-tests:
+
+Style Guide checks
+--------------------
+
+Proselint_  is used for style testing the docs.
+Apart from the inbuilt tests in proselint,
```

Comment:
As a poet, I really like "inbuilt," but it is not standard in American English.
(It is a Britishism.) 
Americans, lacking some grace on this point, use "built-in."